ansibullbot/wrappers/ghapiwrapper.py

```diff
@@ -61,8 +62,13 @@ def get_request(self, url):
         rr = requests.get(url, headers=headers)
         data = rr.json()
 
-        if hasattr(rr, 'links') and rr.links and rr.links.get('next'):
-            _data = self.get_request(rr.links['next']['url'])
+        # handle ratelimits ...
+        if isinstance(data, dict) and data.get(u'message'):
+            if data[u'message'].lower().startswith(u'api rate limit exceeded'):
+                raise RateLimitError()
```

Comment:
@jctanner it looks like GraphQL reports `rateLimit->resetAt` https://developer.github.com/v4/guides/resource-limitations/#returning-a-calls-rate-limit-status and v3 API has `X-RateLimit-Reset`. It'd be better to embed this info into `RateLimitError` exception for more precise delays.